content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+info and comment on this known issue, head to
+[Issue #3375 here.](https://github.com/iterative/dvc/issues/3375) Your input is
+ALWAYS valuable to us! 🙏🏼
+
+### [If I have two cloud folder links added to the DVC config, I'm able to push the data to the default one. How could I push the data to the other cloud folder?](https://discord.com/channels/485586884165107732/563406153334128681/833176227762274364)
+
+You're looking for the `-r / --remote` option for `dvc push`. The command looks
+like this:
+
+```
+dvc push --remote name_of_remote_storage
+```
+
+This will upload tracked files or directories to remote storage based on the
+current _dvc.yaml_ and _.dvc_ files.
+
+### [What's the current recommended way to automate hyperparameter search when using DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/829803720190590986)
+
+Take a look at the new
+[experiments feature](https://dvc.org/doc/start/experiments)! It enables you to
+easily experiment with different parameter values.
+
+You could script a grid search pretty easily by queueing an experiment for each
+set of parameter values you want to try. For example,
+`dvc exp run --queue -S alpha={alpha},beta={beta}`, and then compare the results
+with `dvc exp show`.
+
+We are working on developing experiments to have features or documented patterns
+explicitly for grid search support, so definitely share any feedback to help
+drive the future direction of that!
+
+### [How can I pass sensitive strings (connectionString, containerName, etc) to DVC get without storing them to Git?](https://discord.com/channels/485586884165107732/563406153334128681/830021022337073185)
+
+There's a bit of context behind this question that might give it more meaning.
+
+---
+
+I set up a private GitHub repo to be a data registry and I have set up a private
+Azure remote where I have pushed some datasets.
+
+At the moment I set up the Azure remote using the environment variables
+[as the docs explain here](https://dvc.org/doc/command-reference/remote/add) so
+I don't push sensitive information.
+
+I am now trying to read those datasets from another repository
+("my-project-repo"), using `dvc get` (e.g.
+`dvc get git@github.com:data-registry-repo.git path/data.csv`) but I get this
+error:
+
+```bash
+ERROR: failed to get 'path/data.csv' from 'git@github.com:data-registry-repo.git' - Authentication to Azure Blob Storage via default credentials (https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential) failed.
+Learn more about configuration settings at <https://man.dvc.org/remote/modify>: unable to connect to account for Must provide either a connection_string or account_name with credentials!!
+```
+
+---
+
+Generally, there are two ways solve this issue:
+
+- ENV vars
+- setup some options in the `--global, --system` DVC config
+
+Once they removed some old remotes (with the same name) from their project repo
```

Comment:
not sure readers would understand this, it's better to be more like FAQ, not tell the story behind the specific users' problem